source/assets/stylesheets/styleguide/_colours.scss

```diff
@@ -3,14 +3,14 @@
 $proposition-active-nav: #1d8feb;
 
 
-/* Old depricated greys, new things should use the toolkit greys */
-$grey-0: #0b0c0c;
-$grey-1: #171819;
-$grey-2: #2e3133;
-$grey-3: #454a4c;
-$grey-4: #5c6366;
-$grey-5: #8a9499;
-$grey-6: #a1acb2;
-$grey-7: #b8c6cc;
-$grey-8: #DEE0E2;
-$grey-9: #eaedef;
+// Old depricated greys, new things should use the toolkit greys
+// $grey-0: #0b0c0c;  // $black, $text-colour
+$grey-1: #171819;     // Footer links - hover colour
+$grey-2: #2e3133;     // Borders under list items in proposition header
```

Comment:
Why is this `grey-2` different to: https://github.com/alphagov/govuk_frontend_toolkit/blob/28a01aeb691c0935b6ffea3d248ddf2750b1eed5/stylesheets/_colours.scss#L87

For reference, from toolkit:
```sass
// Standard palette, greys
$black: #0b0c0c;
$grey-1: #6f777b;
$grey-2: #bfc1c3;
$grey-3: #dee0e2;
$grey-4: #f8f8f8;
$white: #fff;
```